src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const x = price * days;
```

Comment:
1. [checklist] Make names descriptive, so they explain the purpose of variables and functions.

You can use `RENT_COST` or something else instead of `x`.